test/fixtures/shortcuts/lambda-full.json

```diff
@@ -103,7 +103,7 @@
             "Arn"
           ]
         },
-        "Runtime": "nodejs6.10",
+        "Runtime": "nodejs10.x",
```

Comment:
Let's change this test to `nodejs12.x` to prove that the non-default path works.

Reply:
done